platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -96,22 +404,18 @@ message Roster {
      */
     repeated RosterEntry rosterEntries = 1;
 }
-```
-
-#### RosterEntry
-
-A roster entry will look as follows:
 
-```proto
 /**
  * A single roster entry in the network state.
  *
- * Each roster entry in SHALL encapsulate the elements required
- * to manage node participation in the Threshold Signature Scheme (TSS).<br/>
  * All fields are REQUIRED.
  */
 message RosterEntry {
 
+    // The tssEncryptionKey is no longer in the roster.
```

Comment:
Is this permanently removed or just moved to another proposal?

Reply:
It is removed from the roster, but still needs to exist.  We'll need to keep the public keys in the state as part of the TssBaseService data.   And if we automate it, we'll need a transaction that publishes the public key to update the table.